nifi-nar-bundles/nifi-hadoop-bundle/nifi-hdfs-processors/src/main/java/org/apache/nifi/processors/hadoop/AbstractHadoopProcessor.java

```diff
@@ -177,9 +178,17 @@ protected KerberosProperties getKerberosProperties() {
         return results;
     }
 
+    @Override
+    public void onPropertyModified(PropertyDescriptor descriptor, String oldValue, String newValue) {
```

Comment:
Alrighty, no problem.  I've added a commit to this PR to remove this onPropertyModified method.  Hopefully the two commits can be squashed to one.  Thanks!